charts/fission-all/templates/router/deployment.yaml

```diff
@@ -35,6 +35,23 @@ spec:
         command: ["/fission-bundle"]
         args: ["--routerPort", "8888", "--executorUrl", "http://executor.{{ .Release.Namespace }}"]
         env:
+        {{- if .Values.authentication.enabled }}
+        - name: ROUTER_USERNAME
```

Comment:
Should we prefer `AUTH_USERNAME` and `AUTH_PASSWORD` ?

Reply:
Fixed